Sources/REventLogger/Networking/NewtorkManager.swift

```diff
@@ -0,0 +1,48 @@
+import Foundation
+
+class NetworkManager {
```

Comment:
none of this is public so it won't be accessible after importing `RSDKUtils`. Is that intended?

Reply:
Yes, as of now this should only be used by event Logger classes